doc/contributing/maintaining-openssl.md

```diff
@@ -55,43 +55,11 @@ NASM version 2.11.08
 
 ## 1. Obtain and extract new OpenSSL sources
 
-Get a new source from <https://github.com/quictls/openssl/tree/OpenSSL_1_1_1j+quic>
+Get a new source from <https://github.com/quictls/openssl/tree/openssl-3.0.5+quic>
```

Comment:
Small nit, maybe we should update the example commit messages later on to match this version for consistency?